neo/public/javascripts/map/controllers.js

```diff
@@ -1,6 +1,9 @@
 angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
   .controller('MapCtrl', function($scope, $window, $http, $compile, Asterix, leafletData) {
     $scope.result = {};
+    $scope.doNormalization = false;
+    $scope.upscaleFactor = 1000 * 1000;
+    $scope.upscaleFactorText = "M";
```

Comment:
For scaleFactor and scaleFactorText, I think it's better to have a global config file or class for us to manage the demo more easily?